prow/plugins/trigger/pr.go

```
@@ -82,6 +86,7 @@ I'm waiting for a [%s](https://github.com/orgs/%s/people) member to verify that
 
 	owner := pr.Base.Repo.Owner.Login
 	name := pr.Base.Repo.Name
+	ghc.AddLabel(owner, name, pr.Number, needsOkToTest)
```

Don't ignore the error!